### DISCUSSION TITLE
2024-03-22 - How to Disagree with Someone More Powerful - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by piers-sinclair** [SSWConsulting/SSW.Rules.Content#8200] [Chewing the Fat]
Hey SSWers,

Working with senior colleagues is an art, but there's lots of tips and tricks you can use to make it easier.

Discuss your ideas 💡 below.

https://ssw.com.au/rules/disagreeing-with-powerful-stakeholders/

**Comment by tiagov8** [comment]
From the tips the most valuable in my opinion: 
- Keep focused on the main goal (common target)... It's easy to lose focus when looking at complex applications, too many variables
- Base things on facts/logic... that should be true for any decision making
- Avoid bad adjectives... probably one of the hardest to follow, but important as it avoid people to take things personally

**Comment by BrookJeynes** [comment]
Being able to understand where the other person is coming from is huge in a disagreement.

**Reply by kikibianc** [reply]
Totally agree

**Reply by tomek-i** [reply]
basically keep an open mind and listen and shove your ego to the side :-D

**Comment by brydeno** [comment]
I've always found that once we shift the discussion to risks and benefits, then a decision is always easier.
As such it is enormously useful to be able to clearly state risks and benefits of various approaches objectively.

**Comment by bahjatalaadelssw** [comment]
Handle disagreements ASAP given that you are emotionally neutral, when your memory is sharp. Acknowledge the other person's thought process and stay solution oriented.

**Comment by AlexRothwell** [comment]
I find that ensuring that both parties share an understanding of the context around the disagreement is important. These tips help to arrive at a common understanding so the best decision for the situation can be found

Remember that the stakeholder may know something which you don't and it may be driving their decision. Keeping the conversation calm and rational will often give them the opportunity to share so you can understand the eventual decision better

**Comment by 2fernandez** [comment]
Ha ha ha ha....

**Comment by MarcusIrmscherSSW** [comment]
Great video, full and excellent tips. Always remember to stay respectful and never make it personal.

**Comment by sethdaily** [comment]
Clarifying the goal/finding a common target is Step 1. I think communication is always less efficient if that step is skipped.

**Comment by jakebayliss** [comment]
I dont really understand why someones 'Power' dictates how I disagree. It would likely be the same way no matter who it is. Just be respectful

**Comment by danielmackay** [comment]
While it's important to speak your mind, I also believe you should pick your battles.  Not every disagreement needs to be voiced.  Not every disagreement is important.

**Comment by calumjs** [comment]
More responsibility is borne by the senior person, who should aim to foster an environment where disagreements can be voiced freely

**Comment by alexleessw** [comment]
I think it's important to understand where the person was coming from especially from Senior & leadership roles.

**Comment by cvionnet** [comment]
Both parties should engage in active listening, remain open to the conversation and be ready to admit when their wrong

**Comment by ncn-ssw** [comment]
- Asking permission to disagree is woeful advice - you are a professional who is ethically bound to voice concerns about courses of action, not a doormat.
- Understanding your senior's priorities tends to reduce disagreements in the first place.
- Though rare, sometimes it's not just "your opinion" - there are genuine situations where the proposed course of action will lead to definite detrimental consequences, and you do no one any favours by pretending otherwise. See the history of armed conflict for a smorgasbord of examples.

**Reply by leahy268** [reply]
> Asking permission to disagree is woeful advice - you are a professional who is ethically bound to voice concerns about courses of action, not a doormat

Agreed but it's often worth asking if they are happy to talk about it.  There's 0 point in talking about it if they don't want to listen.

**Reply by Hona** [reply]
Some nice comments pretty relevant from the YouTube video I liked:

> One thing I have done as the head of Communications is said “my job is to protect your reputation and the reputation of the company. With that in mind I am sharing this feedback”

> I’m a VP and I want anyone at any level to disagree with me at any time, and I would do the same with anyone above or below me. Clarity is kindness and if you don’t agree you need to be clear about that. We don’t have time to wait for someone to work up the courage and find the right time. The right time is always now, when the discussion is happening. If we live in fear of raising our voice we put the entire company at risk. Just say what you think and be ready for an ensuing discussion so we can all gain greater clarity.

Its both the 'lower' employees responsibility to voice concerns, same as senior leadership to listen. Now is usually the right time as it can save or make the most money

**Comment by synnoveisabel** [comment]
Be respectful and open for listening to others opinion, no matter of 'Power'.

**Comment by matt-goldman** [comment]
This is an excellent rule and an excellent video. Just need to remember to not take things personally (easier said than done, I will spend the rest of my life trying to improve at this).

And don't forget this rule: https://www.ssw.com.au/rules/for-the-record/. If you don't agree with someone senior, it's their call. Just let it go, and send a "for the record" if you feel you need to.

And most importantly of all, don't forget the DBAD principle.

**Comment by tkapa** [comment]
I dislike the wording on this title, as it implies that these steps can be disregarded when talking to those you don't consider more "powerful". 

Good video, and a good rule.

**Reply by Hona** [reply]
Maybe just - 'Do you know how to disagree with Senior Leadership' etc, although it becomes less broad

**Comment by Calinator444** [comment]
I think it's best when both parties are focused on mutually reaching the best outcome for the project. It might not always be apparent to the other party that this is your goal so I think using language that steers the conversation in the right direction is a great idea.

**Comment by awaltos** [comment]
Good tips laid out well in the video. Disagreements can also just be contributions towards the best solution, which are important. I like the idea of having strong opinions weakly held as is helps to remain calm and detach from taking things personally and personally attacking which kind of sums up several tips in the video.

**Comment by joshbermanssw** [comment]
Disagreements are not fun for either party, efficient outcomes (grounded in logic) are beneficial for most!

**Comment by lukecookssw** [comment]
Great documentary about this topic:

![images](https://github.com/SSWConsulting/SSW.Rules.Content/assets/100659715/e649dc65-9c11-4eda-b5ec-f3746379b5ba)

**Comment by AntPolkanov** [comment]
Great tips!
The advice to "ground yourself in logic" seems obvious, but it may not be easy to implement because people are inherently illogical. If two logical people have a disagreement, the problem is already halfway solved.

**Comment by TobyChurches** [comment]
There should be another rule for talking to people that are 'less powerful' to make sure that the respect is reciprocated 😅
This is a great rule, and the video lays out the course of action very well... but, this isn't a skill that can be picked up by listening to a short video. It is something that needs to be constantly practiced. Its very important to keep the discussion logical.

**Comment by tomek-i** [comment]
When staying calm and logic gets you nowhere then it's time to call out 

**Trial by Combat**

**Comment by Harry-Ross** [comment]
Super important to ensure that disagreements within teams are resolved quickly and painlessly - so a super important topic. 

However, asking permission to disagree to me seems to me like a way to create an unproductive environment where people are fearful to express opinion. It's important that disagreements can be freely communicated, no matter one's role.

**Comment by Ben0189** [comment]
great tips, stay humble😊

**Comment by steven0x51** [comment]
Excellent rule and video. Managing disagreement can be challenging, particularly when doing so in a second language.

**Comment by amankumarrr** [comment]
Addressing disagreements isn't always about immediate resolution. There have been instances where I initially disagreed with certain points, only to later find common ground after understanding the underlying concerns. 

It's essential to voice disagreements, as it fosters open communication and can lead to eventual resolution, even if it takes time to fully comprehend the underlying issues.

**Comment by BenDunkerleySSW** [comment]
It can be infuriating talking to someone that refuses to listen. Its important to recognise when a constructive conversation can be had & when you should try to approach dispute resolution differently. 

Staying calm and composed is crucial in any case. If you let your emotions get the best of you, you suspend your logic and your rationality. 

Hierarchy shouldn't always determine the value of someones input.

**Comment by Dhruv-0987** [comment]
I just think your knowledge of the problem should be rock solid before going head first into an argument 😄, also sometimes people you disagree with you could be circumstantial so don't think too much of it, the person I disagree with the most is my best friend 😆.

**Comment by chrisschultzssw** [comment]
Great video, and an important topic. As others have said, this does not only apply to disagreeing with someone _more powerful_.

**Comment by jeoffreyfischer** [comment]
It's always difficult to know when you should raise a disagreement with a senior - most juniors would feel pressured and would probably not say anything. So knowing how to handle these situations is important as it might help some people to speak up more often and have valuable conversations.

**Comment by leahy268** [comment]
Most of the rule applies to disagreeing with anyone.

- Repeat the problem as you see it
- Ask if now is a good time to talk about it.
- Stick to the logic never the people
- Find a common target
- Avoid judgmental language
- Express that it's your opinion

In fact the only one that I can see that is for someone more powerful is:

- Leave it up to them

**Reply by Hona** [reply]
100%.

**Comment by jimmy1992abc** [comment]
I think the most important thing is to show your respect regardless of the level of employment. To recognise the other's professionalities and exchange opinions rather than to command and order.

**Comment by ricksu978** [comment]
I like avoiding judgmental language, remaining calm throughout the conversation.

**Comment by stephanfakossw** [comment]
I think that as a professional, it is my obligations to speak up if I see something that's wrong. Don't see a reason to ask permission for it...

**Comment by camillars** [comment]
When it comes to speaking your mind, it feels odd having to ask permission to be honest, like the communication isn't completely open. 
However, if something hits you emotionally, I believe it's okay to let it show you are stressed about it, respectfully. I also disagree with 'power' dictating what/how I should think, staying calm and respectful is key for both sides.

**Comment by KahaMason** [comment]
Being respectful and open minded is key. Be firm in your stance but be understanding of other people's perspectives / concerns on the matter. Essentially we are all on the same boat and we all want the best outcome.

**Comment by jackreimers** [comment]
I think it's important to voice your opinions in a respectful manner and have that opinion heard.

I disagree with the point on asking permission to voice your disagreement, it implies your idea has less merit before you even say it. Superiors should always be open to being challenged and discussing issues (respectfully) and if an employee does not feel empowered to speak up I see that as a sign of workplace dysfunction.

**Comment by bettybondoc** [comment]
I make it a point to choose my battles wisely, especially when navigating disagreements.

In the inherently subjective field of design, we often encounter diverse opinions. While established principles guide us, they don't guarantee universal agreement. What appeals to one person may not to another, underscoring the role of design theory as a guiding compass rather than an absolute directive, and promoting a rich diversity of perspectives and solutions within the design community.

Therefore, I only push back when necessary and when I am able to provide constructive feedback. Above all, I prioritise respect in my interactions. I habitually reflect on whether I would feel respected if I were on the receiving end of my own communications — no matter if I'm talking to a senior or junior colleague.

**Comment by JackDevAU** [comment]
Remain calm, be respectful and you'll be able to voice any disagreement! 
Even if it doesn't change anything, let it be known with a "For the record" ⭐

**Comment by jaydenalchin** [comment]
I'm not sure asking permission to disagree is necessary, feedback and discussion is healthy and should be welcomed. 

A lot of these are great tips for handling disagreement in any situation though. Communicate **why** you think/feel a certain way and encourage the other party to convey their perspective in turn. Pick your battles. Be respectful. Be open to having your mind changed. If you don't feel heard, I'm a big fan of [For the record: ](https://www.ssw.com.au/rules/for-the-record) to make sure your perspective is recorded somewhere.

**Comment by MattParkerDev** [comment]
Always keeping an open mind is very important. I've gone into many calls with people with the mindset of they are wrong and I need to change their mind, but I always find out the reasoning for their decision first. A lot of the time, their reasoning makes sense and thats the end of the conversation :)

**Comment by jerryluossw** [comment]
Great video and great rule, I have a different opinion about asking for permission.
When people are discussing something, it should have permission, otherwise why would people start talking about it?
Also, when you say something(e.g. when you repeat your understanding of the issue), you will express your opinions very naturally, it's very strange to ask if you have permission.

**Comment by tino-liu** [comment]
I agree with choosing battles wisely goes a long way. That and humility.
Being able to put yourself in another's shoe helps better understand both parties, that gives you an edge on expressing opposing ideas strategically.

**Comment by Aibono1225** [comment]
I'm also not 100% in agreement with asking for permission, just have a feeling that this would break the pace of whole conversation.
But overall it's important to be respectful to show a good attitude, nice video + rule!

**Comment by jernejk** [comment]
Usually, empathy, understanding and logic are sufficient to resolve most disagreements, but sometimes, not even the tips in the video are enough. The tips are amazing and should be fine for 95% of disagreements.

In my experience, the other 5% come from emotions, personal struggles, and the need for control, which often overshadow logic when disagreeing with someone more powerful or set in their ways. Empathy and patience are crucial, as people deeply invested in their beliefs may not be swayed by rational arguments alone.

Sometimes, when faced with stubbornness or a strong need to 'win'—often stemming from workplace pressures (often not related to the project or the disagreement), insecurities, or personal challenges—it's best to pause the conversation and revisit it later. Be aware that some may resort to psychological tactics or charisma to sway your opinion, not always with genuine intent but sometimes as a strategy to assert dominance or control. Recognizing these dynamics can help navigate these complex interactions more effectively. If you see anyone using mental manipulations to lower people's self-esteem, gaslighting and methods of psychologically aggressive methods of persuasion, you should involve the manager in the conversation to mediate the situation.

🤖🧑‍💻

**Comment by KristenHu** [comment]
Great video, but cliché. Most of the advice has already been heard by many people, but it can still be challenging to argue effectively with others at times...

**Comment by sylhuang** [comment]
I think all these advices apply to having disagreement with someone at the same level, e.g. your friends, or we are expecting the conversation to be a fair game to start with. But I feel that to make this topic actually useful, we need tips and tactics when the game is unfair. If talking to the senior is just like talking to a friend, I would give most credit of a success conversation to the senior 😋

**Comment by SofieHong** [comment]
Love the video. Additionally, be prepared to possibly receive a "No" when we propose our suggestions, and be ready to move on 🏃

**Comment by jimmidier** [comment]
Disagreements are inevitable and will always occur as soon as a second voice comes in. I see it as a natural product of human diversity. 
When it comes to collaboration and team work, this nature will have negative impacts on efficiency, and that's where a "more powerful" role comes into play.

Therefore, I see "someone more powerful" as a solution to disagreements, but not impediments. In which case, by the end of the day it'll come to "Leave it up to them".

For the rest of the tips in the rule, I see them as common tips for general communication, where it's about respecting human nature, and the power from the decision maker is what protects the team from being impacted by the negative side of it.

IMO the rule actually suggests 2 parts of the topic. 
Q: How to disagree (with anyone)?
A: Ground yourself in logic, Avoid judgemental language...

Q: And if there's someone more powerful?
A: Do the same, but leave it up to them

**Comment by babakamyljanovssw** [comment]
Great tips! Being open minded, respectful, leaning towards to a common target, and avoiding the judgemental language are key to any conversation. These tips can be challenging and needs to be practised.

**Comment by Freego1783** [comment]
Nice tips overall, but I think the best tip here is "stay calm", cause if you fail at that you can't really apply the other tips correctly. But it can be hard sometimes depending on who you're having a disagreement with.

**Comment by PothieuG** [comment]
Requesting permission for having a disagreement conversation is a new (and somewhat strange) concept to me. I don't really believe in that. If I need to say something, I will do it respectfully of course, but I won't ask permission.

However, I find the rule great, as it open my perspective on problem-solving.

**Comment by YalalCakirSSW** [comment]
I don't know how to feel about it. It's a good rule, but it's theoretical. I think it requires time, practice, and constant thinking to perform these points. Also, I think it depends on the person with whom you disagree.

**Comment by christoment** [comment]
Great rule overall, but it's a bit sad that we as a human has to resort this approach when exchanging great ideas with another people who have more power.

I don't really like the point 2 "Ask permission" since it feels unnatural to ask permissions in discussion... Whereas the discussion is for the other's person benefit (e.g. PO - it's for their own product).

I also like that this rule is paired with for the record rule in disagreement 😄

**Comment by vladislav-kir** [comment]
Great video and a rule!

**Comment by Levijj22** [comment]
Interesting video. I dont agree with everything she said i.e. getting approval to disagree, however generally I always try not to get caught up on principle and put pride to the side.

**Comment by GordonBeeming** [comment]
The video was great, on a very short CTF, It feels weird for us to cut good context that talks about the dynamics of the company creating space or not for open conversations, i.e. Doing the risk assessment of saying vs not saying something.

Overall a good topic, it's great to help people build their communication toolbox.

**Comment by drwharris** [comment]
I dont like the use of "Asking permission". In these situations, you are providing your expertise and shouldnt need permission to do so. I would probably call it "Providing input". While conversationally you might lead into it with "If I might provide my thoughts" or the likes, you arent really asking permission.

**Comment by leehawkinsssw** [comment]
The video has some decent tips.

We need to be careful not to confuse power and seniority - to me, seniority is more about experience than power. 

It's also worth noting that working out who has more seniority or power is error-prone, so I'd encourage people to worry less about the power/seniority dynamics of a situation and focus more on learning how to communicate effectively no matter the status/power/seniority of the other party. Knowing how to _disagree_ without being _disagreeable_ is the key here.

**Comment by JackDuan123** [comment]
I never have trouble verbally disagree with anybody, so I disagree with this vedio.
We want to disagree with somebody, it is mostly likely we have a lot of judgy adjectives, if we only share facts to those who we want to disagree, it is most likely we are not honest.

One scientific fact:

**We can't think without having emotions :)**

I don't want to be dishonest just to make what I want to happen happen because this is bad and we have to admit that at the end of the day we are human beings, even genius human makes mistakes and a lot of judgy adjective, Yes, I am talking about Albert Einstein, and I think he come up with theory of relativity out of the hate on mechanical determinism.

What I would do: honestly tell the others the facts that I perceive and also tell them my judgy adjectives and my emotions and feelings.

**Comment by tombui99** [comment]
All these tips are super helpful which I will definitely use whenever I have a discussion with someone more powerful. Another important tip that should be added is that we could bring in another senior person into the discussion to help support your opinions.

**Comment by PennyWalker** [comment]
Don’t let conflicts linger. Think about what you’d like to say and approach with an olive branch. Actively listen to the other side to make sure you understand what they’re trying to say. It’s so easy to have a misunderstanding. When needed bring in a 3rd person, they often pick the disconnect before the parties involved and can help then understand the other position.

**Comment by ozdemir-mehmet-cer** [comment]
Ground yourself in logic is probably the best tip.
